virttest/qemu_vm.py

```diff
@@ -992,7 +992,7 @@ def set_value(opt_string, key, fallback=None):
                 else:
                     self.spice_options['spice_port'] = self.spice_port
                     spice_opts.append("port=%s" % self.spice_port)
-            else:
+            elif optget("spice_port") != "no":
```

Comment:
The `no` here is being used on an attempt to be consistent with the values of other parameters (`no` means disabled), right? If so, sure, looks good.